bin/testObservability/helper/constants.js

```diff
@@ -2,7 +2,7 @@ const path = require('path');
 
 exports.consoleHolder = Object.assign({},console);
 exports.BATCH_SIZE = 1000;
-exports.BATCH_INTERVAL = 2000;
+exports.BATCH_INTERVAL = 1000;
```

Comment:
we do not want to reduce this.
- Increase the teardown time of request batcher
- Add more debug logging to track the flow of the events.